packages/exchange-ui-core/src/features/bundles/reducer.ts

```diff
@@ -4,18 +4,30 @@ import { IBundleAction, BundlesActionType } from './actions';
 export interface IBundlesState {
     bundles: Bundle[];
     showBundleDetails: boolean;
+    isLoading: boolean;
 }
 
 const initialState: IBundlesState = {
     bundles: [],
-    showBundleDetails: false
+    showBundleDetails: false,
+    isLoading: false
 };
 
 export function bundlesState<T>(
     state: IBundlesState = initialState,
     { type, payload }: IBundleAction
 ): IBundlesState {
     switch (type) {
+        case BundlesActionType.FETCH_BUNDLES:
+            return {
+                ...state,
+                isLoading: true
+            };
+        case BundlesActionType.FETCH_BUNDLES_SUCCESS:
+            return {
+                ...state,
+                isLoading: false
+            };
```

Comment:
Why don't create a single action for setting the item loading state? 
Like setBundlesLoading(true/false)?